docs/scos/dev/front-end-development/202307.0/oryx/building-applications/styling/oryx-responsive-design.md

```diff
@@ -0,0 +1,215 @@
+
+## Breakpoints
+
+Breakpoints play a crucial role in responsive design, as they define the specific screen sizes at which your website or application layout should adapt. Oryx's Responsive Design system provides default breakpoint for small, medium and large screen sizes. You can adjust the defaults or introduce custom breakpoints to tailor the design system to your specific needs.
+
+Breakpoints are configurable in themes. If you use an Oryx theme it comes with preconfigured breakpoints, but you can override the breakpoints in an additional theme configuration.
+
+{% info_block infoBox "" %}
+
+You'd assume that breakpoints and their usage would be controlled by design tokens, and used as CSS variables. Unfortunately, CSS variables can not be used inside media queries definitions. This is the main reason why Oryx provides a configurable system to provide the breakpoint definitions.
+
+{% endinfo_block %}
+
+### Default Breakpoints
+
+Oryx provides the following default breakpoints:
+
+| Breakpoint | Size Range (in pixels) |
+| ---------- | ---------------------- |
+| `Size.sm`  | Up to 767              |
+| `Size.md`  | 768 - 1023             |
+| `Size.lg`  | 1024 and above         |
+
+The default breakpoints are given by the `defaultBreakpoints` object, available in the [themes](https://www.npmjs.com/package/@spryker-oryx/themes) package. The breakpoint values are configured in pixels.
+
+```ts
+import { Breakpoints, Size } from "@spryker-oryx/utilities";
+
+export const defaultBreakpoints: Breakpoints = {
+  [Size.Sm]: {
+    max: 767,
+  },
+  [Size.Md]: {
+    min: 768,
+    max: 1023,
+  },
+  [Size.Lg]: {
+    min: 1024,
+  },
+};
+```
+
+The breakpoint definition shows that you only need to define the required breakpoint min or max values. For example, there's no `min` value provided for small size, as it will default to `0`. The same is true for the large screen, which starts with 1024 and will support all screens that are larger than the 1024 px.
+
+### Custom Breakpoints
+
+If the default breakpoints do not align with your project's requirements, Oryx allows you to define custom breakpoints. This gives you the flexibility to adapt your design to specific devices or screen sizes that are relevant to your target audience.
+
+Breakpoints are part of Oryx themes. Oryx themes use the default configuration, but you can configure a(n additional) theme with custom breakpoints.
+
+```ts
+import { Size } from "@spryker-oryx/utilities";
+
+export const app = appBuilder()
+  // ...
+  .withTheme([storefrontTheme])
+  .withTheme({
+    breakpoints: {
+      [Size.Xs]: {
+        max: 575,
+      },
+      [Size.Sm]: {
+        min: 576,
+        max: 767,
+      },
+    },
+  })
+  .create();
+```
+
+In this example, we have defined custom breakpoints for extra-small (XS), small (SM), medium (MD), large (LG), and extra-large (XL) screen sizes. You can adjust the minimum and maximum values to suit your specific design requirements.
```

Comment:
I would use only shortened versions of sizes (xs, lg, ...), and need to keep the same case of the size codes for the whole doc, ie lower, upper or capitalised

And also, the sentence contains mention of `md`, `lg` and `xl` custom breakpoints, but example override only `xs` and `sm`, not very clear that we are providing one new and overriding the existing one. `xl` one even is not used by default

Reply:
yup, those are not in sync indeed.